app/views/shelter_reviews/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<h2>Update Review:</h2>
+<section class="form">
+  <%= form_tag("/shelters/#{@shelter_id}/reviews/#{@review.id}", method: "patch") do %>
+    <%= label_tag(:title, "Title: ") %>
+    <%= text_field_tag(:title, value = @review.title) %><br />
+    <%= label_tag(:rating, "Rating: ") %>
+    <%= select_tag(:rating, options_for_select(["1", "2", "3", "4", "5"])) %><br />
```

Comment:
Mike did say something negative about this drop down menu, and I'll delete it if you think we should. It seems more complicated to try and limit the valid input from users to numbers 1-5 with a flash message or something.

Reply:
I love the drop down! Please don't remove.

Reply:
yay!